packages/scene-composer/package.json

```diff
@@ -159,6 +159,7 @@
     "path-browserify": "1.0.1",
     "postprocessing": "6.28.5",
     "react-color": "^2.19.3",
```

Comment:
If we are not using react-color which is lacking the a11ly. Then better to remove from package.json?

Reply:
I left the CirclePicker from react-color in tag-style/ColorPicker/ColorPicker.tsx since it is accessible via keyboard. I just changed the 'custom color' selection from the SketchPicker to this custom one.

Reply:
I see in this new color library, we would have to explicitly click on the color picker to save the selected color. We want to customer to just click on the color and that should reflect as chosenColor.  Is it possible to make save optional?